services/rewards/service.go

```diff
@@ -168,3 +179,15 @@ func (s *Service) GetParameters(ctx context.Context, currency *BaseCurrency) (*P
 
 	return params, nil
 }
+
+type CardBytes []byte
+
+func (s *Service) GetCardsAsBytes(ctx context.Context) (CardBytes, error) {
+	out, err := s.s3Svc.GetObject(ctx, &s3.GetObjectInput{Bucket: &s.cfg.Cards.Bucket, Key: &s.cfg.Cards.Key})
+	if err != nil {
+		return nil, err
+	}
+	defer func() { _ = out.Body.Close() }()
+
+	return io.ReadAll(out.Body)
```

Comment:
We are reading from an S3 bucket we own.

Reply:
This one seems like it could be a false positive since I believe this is a GET endpoint. Couldn't hurt to do this as a defense in depth measure though.

Reply:
We tend to use a LimitReader when dealing with  [external requests](https://github.com/brave-intl/bat-go/blob/81d63db3201fb6a82a1cc93463899c1207bea09f/services/skus/controllers.go#L1231) but in places where we are calling services we own (in this case the S3 bucket which we put the contents in) we haven't been adding it hence the decisions to not add it here. 

Added it in this PR https://github.com/brave-intl/bat-go/pull/2695/files